aws/resource_aws_default_security_group.go

```diff
@@ -87,21 +236,111 @@ func resourceAwsDefaultSecurityGroupCreate(d *schema.ResourceData, meta interfac
 		return fmt.Errorf("Error finding default security group: no matching group found")
 	}
 
-	d.SetId(*g.GroupId)
+	d.SetId(aws.StringValue(g.GroupId))
 
 	log.Printf("[INFO] Default Security Group ID: %s", d.Id())
 
 	if v := d.Get("tags").(map[string]interface{}); len(v) > 0 {
 		if err := keyvaluetags.Ec2CreateTags(conn, d.Id(), v); err != nil {
-			return fmt.Errorf("error adding EC2 Default Security Group (%s) tags: %s", d.Id(), err)
+			return fmt.Errorf("error adding EC2 Default Security Group (%s) tags: %w", d.Id(), err)
 		}
 	}
 
 	if err := revokeDefaultSecurityGroupRules(meta, g); err != nil {
-		return fmt.Errorf("%s", err)
+		return err
+	}
+
+	return resourceAwsDefaultSecurityGroupUpdate(d, meta)
```

Comment:
Not for this PR but we should get these two resources switched from calling Create -> Update and instead Create -> Read. 👍